### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by haymac** [PawelGorny/MinikeyCuda#1]
Hi,

i use this config : 
MinikeyCuda.exe -b 256 -t 256 -rangeStart S32222222222222AYhJn9z -input serie1.txt
loaded: 4908
Using:
NVIDIA GeForce RTX 3090 (82 procs)
maxThreadsPerBlock: 1024

program found : 
1F6RfFuPsE9wqjMP79QMqjxztLGfyfh4tk
6798DC94F513F3BFF59DDF2F402A695CAE4EE86D13BF80D7DE3B79F9558D474D

but 1F6RfFuPsE9wqjMP79QMqjxztLGfyfh4tk not in serie1.txt.

**Comment by PawelGorny** [comment]
Yes, I use bloom filter which gives false positive results. I set it that chance to get false positive is very very low, you had (bad) luck ;) anyway, you may always check manually. Later I will add double check, now it is not the most important.

**Comment by kuw13** [comment]
give me alot off false addresses 
can you fix the bloom filter please .
good work
